app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -100,10 +106,22 @@ protected void onCreate(Bundle savedInstanceState) {
             if (Globals.CurrentMatchNumber < Globals.MatchList.size()) {
                 Matches.MatchRow Match = Globals.MatchList.getMatchInfoRow(Globals.CurrentMatchNumber);
                 if (Match != null) {
-                    // TODO Set "Teams" to the options in the single select dropdown
-                }
+                    int[] Teams = Match.getListOfTeams();
+                    String[] teamsInMatch = new String[Teams.length];
+                    for (int i = 0; i < Teams.length; i++) {
+                        teamsInMatch[i] = String.valueOf(Teams[i]);
+                    }
+                    ArrayAdapter<String> adapter = new ArrayAdapter<String>(this, R.layout.cpr_spinner, teamsInMatch);
```

Comment:
I assume we can't have an ArrayAdapter\<int\> ??
We don't do this often (1x per match) so no biggie, but if we really need a String[], we should have a class function called "getStringListOfTeams" that returns String[].  The class can store 6 extra members as String str_red1, etc.  (so the class doesn't have to computer them over and over again).

Also, if this is the ONLY place getListOfTeams is called, then just convert this to return String[].  Perhaps we can even store them in the Class as String if no one else uses it.